example/openshift/openshift_deploy.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+echo DEPLOYING ECLIPSE HONO ON OPENSHIFT
+
+# creating new project
+oc new-project hono --description="Open source IoT connectivity" --display-name="Eclipse Hono"
+
+# creating the directory for Hono Server persistent volume
+if [ ! -d /tmp/hono ]; then
+    mkdir /tmp/hono
+    chmod 777 /tmp/hono
+else
+    echo /tmp/hono already exists !
+fi
+
+# creating Hono Server persistent volume (admin needed)
+oc login -u system:admin
+oc create -f ../../application/target/fabric8/hono-app-pv.yml
```

Comment:
since most of these paths seem to refer to the `applicaiton` module anyway, wouldn't it make more sense to put this script into that module in the first place?